src/components/App.tsx

```
@@ -44,6 +45,10 @@ const App: React.FC<AppProps> = (props) => {
     return <StreamContainer />
   }
 
+  if (small || mobile) {
+    return <MobileContainer />
+  }
```

Now that we manage the mobile case from here, I think you should also remove the throw in unity-renderer to prevent unexpected behaviors from kernel and renderer

---

I'm counting on that throw to stop unity. How do you think that could affect?

---

@menduz ?

---

We are thinking on removing that throw at some point. Shouldn't be a blocker for this PR